charts/telepresence/templates/excluded-variables-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if not .Values.rbac.only }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: telepresence-excluded-variables
+  namespace:  {{ include "traffic-manager.namespace" . }}
+data: {{ range .Values.variables.excluded }}
```

Comment:
Even if it is a long name, I'd suggest using `.Values.intercept.environment.excluded`. It's more concise and will also be extendable if we want to add more functionality in the future (substitutions, default values, etc.).